echo/server/tests/common.py

```
@@ -35,6 +36,7 @@
 
 
 def delete_conversation(conversation_id: str):
+    logger.debug(f"Deleting conversation: {conversation_id}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)